interpreter/mesh/src/main/scala/io/buoyant/interpreter/MeshInterpreterInitializer.scala

```diff
@@ -0,0 +1,109 @@
+package io.buoyant.interpreter
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.conversions.time._
+import com.twitter.finagle._
+import com.twitter.finagle.buoyant.{H2, TlsClientPrep}
+import com.twitter.finagle.naming.NameInterpreter
+import com.twitter.finagle.service.Backoff
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.logging.Logger
+import io.buoyant.namer.{InterpreterConfig, InterpreterInitializer}
+import io.buoyant.interpreter.mesh.Client
+import scala.util.control.NoStackTrace
+
+/**
+ * The namerd interpreter offloads the responsibilities of name
+ * resolution to the namerd service via the namerd streaming gRPC API.
+ * Any namers configured in this linkerd are not used.
+ */
+class MeshInterpreterInitializer extends InterpreterInitializer {
+  val configClass = classOf[MeshInterpreterConfig]
+  override def configId: String = "io.l5d.mesh"
+}
+
+object MeshInterpreterInitializer extends MeshInterpreterInitializer
+
+object MeshInterpreterConfig {
+  private val log = Logger.get(getClass.getName)
+
+  val DefaultRoot = Path.Utf8("default")
+
+  val defaultRetry = Retry(1, 10.minutes.inSeconds)
+}
+
+case class Retry(
+  baseSeconds: Int,
+  maxSeconds: Int
+)
```

Comment:
I'd prefer to unify this with [JitteredBackoffConfig](https://github.com/linkerd/linkerd/blob/master/linkerd/core/src/main/scala/io/buoyant/linkerd/Router.scala#L265).